tests/net/socket/tcp/src/main.c

```diff
@@ -1075,7 +1183,10 @@ void test_main(void)
 		ztest_unit_test(test_v6_so_rcvtimeo),
 		ztest_unit_test(test_v4_msg_waitall),
 		ztest_unit_test(test_v6_msg_waitall),
-		ztest_user_unit_test(test_socket_permission)
+		ztest_user_unit_test(test_socket_permission),
+		ztest_unit_test(test_v4_send_recv_block),
+		ztest_unit_test_setup_teardown(test_v4_send_recv_block,
```

Comment:
The test is added twice, is that intended?

Reply:
Yes, the first one is without packet loss, the second one with